CMakeLists.txt

```diff
@@ -39,6 +39,16 @@ if(WITH_TBB)
     add_definitions(-DWITH_TBB)
 endif()
 
+# OpenMP support
+set(WITH_OMP OFF CACHE BOOL "use OpenMP for on-node threading" )
```

Comment:
If the user specifies something different (i.e. THREADING_MODEL=foo) We load serial or just throw and error?

PD: If THREADING_MODEL is not specified, serial is the default model